bot.py

```diff
@@ -377,6 +407,7 @@ async def stop(self, message, _command_content):
 
         self._stop()
         logging.info("Stopped media for user %s", message.author)
+        await self.attempt_disconnect()
```

Comment:
Why not `self.loop.create_task(self.attempt_disconnect())`?

Reply:
good point